openshift/templates/issuer-agent/issuer-agent-deploy.prime-sandbox.dev.param

```diff
@@ -3,20 +3,21 @@
 # Component: .
 # Template File: ../openshift/templates/issuer-agent/issuer-agent-deploy.yaml
 #=========================================================
-# NAME=agent
+# NAME=agent-sandbox-prime
 # APP_NAME=issuer-prime
```

Comment:
In this case, this is to use the same template as other agents, and being able to group this with the other PRIME services, so it is fine in this scenario 👍🏻